### PR TITLE
Reformat all files with prettier 2.0

### PR DESCRIPTION
**Pull request by michalczaplinski**
#### Description of what you did:

`npx prettier --write "packages/**/*.{js,jsx,ts,tsx}"`

Plus fixed 2 unrelated eslint complaints.

#### My PR is a:

- 🔝 Improvement

#### Is the PR ready to be merged?

- ✅ Yes!

### CODE REVIEW

**Comment by michalczaplinski** [comment]
@luisherranz Should I add a changeset for all the packages...? 🤔 😅

**Comment by luisherranz** [comment]
Functionality doesn't change so nope.

File: packages/connect/src/__tests__/batching.tests.jsx.rej [hunk 1]
```diff
@@ -0,0 +1,33 @@
+diff a/packages/connect/src/__tests__/batching.tests.jsx b/packages/connect/src/__tests__/batching.tests.jsx	(rejected hunks)
```
Comment (luisherranz) [inline packages/connect/src/__tests__/batching.tests.jsx.rej]: What's this file?

File: packages/connect/src/__tests__/types.tests.tsx.rej [hunk 2]
```diff
@@ -0,0 +1,33 @@
+diff a/packages/connect/src/__tests__/types.tests.tsx b/packages/connect/src/__tests__/types.tests.tsx	(rejected hunks)
```
Comment (luisherranz) [inline packages/connect/src/__tests__/types.tests.tsx.rej]: Same diff file here.

**Review by luisherranz — COMMENTED** [review]
There are a couple of files that I don't know where they came from. May I know?

The rest is fine 🙂

**Review by luisherranz — APPROVED** [review]
Actually, I'm sorry.... I'm going to approve and do whatever you need to do with those files 👍

**Comment by michalczaplinski** [comment]
Yeah, I'm not sure why those were generated, because I didn't merge any branches locally, but they can just be deleted.

**Comment by luisherranz** [comment]
Where do they come from? (I'm just curious)

**Pull request merged**